docs/contributing/README.md

```diff
@@ -1,42 +1,241 @@
+- `ci`: Changes to our CI configuration files and scripts (examples: GitHub, Jenkins).
+- `docs`: Documentation only changes.
+- `feat`: A new feature.
+- `fix`: A bug fix.
+- `perf`: A code change that improves performance.
+- `refactor`: A code change that neither fixes a bug nor adds a feature.
+- `test`: Adding missing tests or correcting existing tests.
+
+#### Scope
+
+We use the "package name" and "ES6 module name" rule, as follows:
+
+**Package rule:**
+
+When the change affects one of these next packages:
+
+- `mathtype-ckeditor4`
+- `mathtype-ckeditor5`
+- `mathtype-froala`
+- `mathtype-froala3`
+- `mathtype-generic`
+- `mathtype-tinymce`
+- `mathtype-tinymce4`
+- `mathtype-tinymce5`
+
+**Modules:**
+
+If the affected package is `mathtype-html-integration-devkit`, use the ES6 module name, instead:
+
+- `accessibility`
+- `backwardslib`
+- `configuration`
+- `constants`
+- `contentmanager`
+- `core`
+- `customeditors`
+- `editorlistener`
+- `event`
+- `global`
+- `image`
+- `integrationmodel`
+- `jsvariables`
+- `latex`
+- `listeners`
+- `mathml`
+- `md5`
+- `modal`
+- `parser`
+- `polyfills`
+- `popupmessage`
+- `serviceprovider`
+- `stringmanager`
+- `telemetry`
+- `test`
+- `textcache`
+- `util`
+
+There are usually a few exceptions to the above rules:
+
+- `demo`: used when updating the integrations examples on the `demos` folder.
+
+- `packaging`: used for changes that change the npm package layout in all of our packages, e.g. public path changes, package.json changes done to all packages, d.ts file/format changes, changes to bundles, etc.
+
+- `changelog`: used for updating the release notes in CHANGELOG.md
+
+- `dev-infra`: used for dev-infra related changes within the directories /scripts and /tools
+
+- _none/empty string_: useful for `test` and `refactor` changes that are done across all packages (e.g. `test: add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `docs: fix typo in tutorial`).
+
+#### Summary
+
+Use the `<summary>` field to provide a succinct description of the change:
+
+- use the imperative, present tense: "change" not "changed" nor "changes".
+- don't capitalize the first letter.
+- no dot (.) at the end.
+
+### Commit Message Body
+
+Explain the motivation for the change in the commit message `body`.
+
+Just as in the `<summary>`, use the imperative, present tense: "fix" not "fixed" nor "fixes".
+
+This commit message should explain _why_ you are making the change.
+You can include a comparison of the previous behavior with the new behavior in order to illustrate the impact of the change.
+
+### Commit Message Footer
+
+The `footer` can contain information about breaking changes and deprecations and is also the place to reference GitHub issues, Kanbanize card, Jira tickets, and other PRs that this commit closes or is related to.
+
+In the case of Kanbanize, it is compulsory to add `#taskid <card_number>` to allow kanbanize to track commits.
```

Comment:
For what i've read, in order to have a consistent footer with conventional commits, "each footer MUST consist of a word token, followed by either a :<space> or <space>#", for example:
* Task-id: 12345
* Task-id #12345

Shouldn't we follow this specification in order to have a footer that conventional  commits can undestand?

Reply:
Thanks for your feedback, Carla.
Not sure If I understand your question, though. :D 

Let's see.

On our model, I was proposing to keep the `#taskid` for backward compatiblity.  

On the example, I provided with the document, the example contains both tickets from GitHub, (public tool), and Kanbanize, (private tool), like:

```
BREAKING CHANGE: <breaking change summary>
<BLANK LINE>
<breaking change description + migration instructions>
<BLANK LINE>
<BLANK LINE>
Fixes #<issue number>
#taskid 99999
```

Or maybe, Is it `"task-id:"` the right way, and not `"taskid:"`? 🤔 

Just to make sure, I've just checked the current guideline, on this very same repo and this PR upgrades, and states this:

```
Present-tense summary under 50 characters

More detailed explanatory text, if necessary.  

- More information about commit (under 72 characters).
- More information about commit (under 72 characters).

#taskid $card_number

See: https://github.com/wiris/integrations-guides/tree/main/git

```

Would it be better to change the examples to this?

```
BREAKING CHANGE: <breaking change summary>
<BLANK LINE>
<breaking change description + migration instructions>
<BLANK LINE>
<BLANK LINE>
Fixes #<issue number>
#taskid $card_number
```

Reply:
I see what you suggest, to clarify things, my question was related to the '#' symbol at the beginning of the footer, i'm not sure if including a '#' at the beginning could cause some misunderstandings, since i haven't seen on the documentation that the conventional commits support additional symbols at the beginning of the footer. 
Nonetheless the last example you purpose es more clear to me since it includes two footers, one for the taskid and one for the issue id.

Reply:
Thanks!